docs/notes.tex

```diff
@@ -564,6 +564,60 @@ \subsection{Ordering of basis functions}
 \verb|GeneralizedContractionShell|
 and overwrite these properties with the desired ordering.
 
+\subsection{Conventions for ordering Cartesian and spherical components}
+\label{sec:angmom_components_ordering}
+
+\paragraph{Cartesian components}
+A shell of generalized contractions of angular momentum $\ell$ is composed of $(\ell + 1)(\ell + 2)$ Cartesian components of the form $x^{a_x}y^{a_y}z^{a_z}$ such that $a_x + a_y + a_z = \ell$. In \verb|gbasis|, the ordering of Cartesian components is stored as an $\ell \times 3$ \verb|numpy| array, where each entry $(i, j)$ corresponds to the exponent $a_j$ for the $i$\textsuperscript{th} component. In other programs, like \verb|HORTON2|, a different notation is used, where the components are written as strings of X, Y, and Z, where the number of times each letter appears corresponds to the exponent $a$ for that letter. These notations are easily convertible, for example, the \verb|gbasis| notation for the component \verb|XXXZZ| is $(3, 0, 2)$
```

Comment:
I'm not sure what the convention is on wrapping latex, but it might not be a bad idea to separate these sentences into multiple lines. In latex, only double newlines start a new paragraph,

Reply:
I was writing this in Overleaf and it wraps lines automatically, so I didn't notice. Would definitely be better to break up the long lines.